Makefile

```diff
@@ -0,0 +1,80 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# See https://packaging.python.org/en/latest/tutorials/installing-packages/
+# See https://packaging.python.org/tutorials/packaging-projects/
+# python3 -m ensurepip
+# python3 -m pip install --upgrade pip setuptools wheel
+# python3 -m pip install --upgrade build
+# python3 -m pip install --upgrade pylint
+
+default:
+	@echo Nothing to make
+
+################################################################
+# Run pylint over the package
+
+pylint:
+	make -C src/cbmc_starter_kit pylint
+
+################################################################
+# Build the distribution package
+
+build:
+	python3 -m build
+
+unbuild:
+	$(RM) -r dist
+
+################################################################
+# Install the package into a virtual environment in development mode
+#
+# Note: Editable installs from pyproject.toml require at least pip 21.3
+
+VENV = /tmp/cbmc-starter-kit
+develop:
+	python3 -m venv $(VENV)
+	$(VENV)/bin/python3 -m pip install --upgrade pip
+	$(VENV)/bin/python3 -m pip install -e .
+	@ echo
+	@ echo "Add $(VENV)/bin to PATH with 'export PATH=\$$PATH:$(VENV)/bin'"
+	@ echo
```

Comment:
Added activation instructions (but notice this export puts VENV at end of PATH so it won't override other pythons in PATH).

Reply:
Thanks Mark.

> but notice this export puts VENV at end of PATH so it won't override other pythons in PATH

That's true yes. I find it this a bit concerning though. Binaries installed in the virtual environment which might have a different interpretation of the environment during installation in the virtual env (packages in the virtual env) vs during execution (the global packages)?
Not super sure, but I know that the activation scripts adds VENV paths at the beginning of the PATH so that the global python is completely shadowed until deactivation.